web/src/main/java/org/mskcc/oncotree/api/GlobalControllerExceptionHandler.java

```diff
@@ -44,7 +44,7 @@
 
     private static final Logger logger = LoggerFactory.getLogger(GlobalControllerExceptionHandler.class);
 
-    @ResponseStatus(code = HttpStatus.INTERNAL_SERVER_ERROR, reason = "Failed to connect to TopBraid")
+    @ResponseStatus(code = HttpStatus.SERVICE_UNAVAILABLE, reason = "Failed to connect to TopBraid")
```

Comment:
We probably really shouldn't be changing the behavior of the API here, but OK.

Reply:
I guess it was part of the commit I had done.

Reply:
I'll revert this to the previous status code (INTERNAL_SERVER_ERROR) now.